src/webgpu/shader/validation/shader_io/interpolate.spec.ts

```diff
@@ -86,4 +86,26 @@ g.test('require_location')
 
 g.test('integral_types')
   .desc(`Test that the implementation requires @interpolate(flat) for integral user-defined IO.`)
-  .unimplemented();
+  .params(u =>
```

Comment:
Note for myself: Earlier tests cover other cases like negative conditions, i.e. `@interpolate(flat)` isn't in places where it shouldn't be. This test is just for `@interpolate(flat)` being present when it is required.